examples/simple-game-server/fleet-distributed.yaml

```diff
@@ -0,0 +1,49 @@
+# Copyright 2020 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# special fleet for load testing where we scale the fleet up/and down
+# it uses artificially low resource requests to be able to put as many
+# pods on a node as possible.
+
+apiVersion: "agones.dev/v1"
+kind: Fleet
+metadata:
+  name: simple-game-server-distributed
+spec:
+  replicas: 0
+  scheduling: Distributed
+  template:
+    spec:
+      ports:
+      - name: default
+        containerPort: 7654
+      template:
+        spec:
+          containers:
+          - name: simple-game-server
+            image: gcr.io/agones-images/simple-game-server:0.1
+            env:
+            # Disables the UDP listener (Enabled by default)
```

Comment:
I  think on all of these, let's default to UDP being enabled, since that's the most common use case. WDYT?

Reply:
Agreed, will do